### PR TITLE
Mh pundit policies

### PR DESCRIPTION
**Pull request by merrillhuang**
Creating a pull request, ignore for now. Will update after industrial auth branch is merged to make a clean commit history (removing commits from industrial auth branch)

### CODE REVIEW

File: app/policies/follow_request_policy.rb
Comment (Ritumka) [inline app/policies/follow_request_policy.rb:1]: You did authorization for everything ,cool!

File: app/policies/user_policy.rb
Comment (Ritumka) [inline app/policies/user_policy.rb:36]: Policies look good!

**Review by Ritumka — APPROVED** [review]
Good job!

**Review by armstrca — CHANGES_REQUESTED** [review]
Alice does not seem to be able to delete her own comments, though she can create/edit/update them.

**Comment by merrillhuang** [comment]
> Alice does not seem to be able to delete her own comments, though she can create/edit/update them.

Made change to allow deleting user's own comments. Removed checking is_an_authorized_user on destroy comments. Allows comment destroy to be authorized using pundit policies instead.

**Review by armstrca — APPROVED** [review]
Looks good. Don't merge this before you merge the previous branch, though.

**Pull request merged**
